tests/sentry/api/serializers/test_apitoken.py

```diff
@@ -0,0 +1,49 @@
+from sentry.api.serializers import ApiTokenSerializer
+from sentry.testutils.cases import TestCase
+
+
+class TestApiTokenSerializer(TestCase):
+    def setUp(self) -> None:
+        self._user = self.create_user()
+        self._scopes = ["test_scope"]
+        self._token = self.create_user_auth_token(user=self._user, scope_list=self._scopes)
```

Comment:
Sorry I realize I should've defaulted to empty scopes in `create_user_auth_token` if it wasn't relevant to the test
```suggestion
        self._token = self.create_user_auth_token(user=self._user, scope_list=[])
```

Reply:
That's okay, perfectly fine to be explicit about having empty scopes. Is that even a feasible token? Does that equate to unbounded (i.e. all permissions)?

Reply:
We should probably add more tests here that make sure scopes are also returned correctly, so it's okay to keep a not empty list. This area in general needs more coverage, but something to follow up on!

Reply:
We do allow empty scopes, the token would just be useless.

Reply:
Is that something we should change?

Reply:
hmm that's a good question. The only use case I can think of for having empty scopes is we allow users to create internal integrations without scopes. They're not required when you follow the `creation flow`.

I'm not super familiar with how people use integrations, or what you can do without a token/no scopes token. There might be a use case there

Reply:
For user tokens, we don't allow it in the UI and you can't create user tokens through other tokens (which is a recent-ish change) so it should be fine